aws/camunda/java/api/src/main/java/io/gravicore/loc/Application.java

```diff
@@ -0,0 +1,34 @@
+package io.gravicore.loc;
```

Comment:
@darlanmoraes Should there be a reference to LOC?

Reply:
@darlanmoraes Perhaps change `.loc` to `.camunda`?

Reply:
@jeremyodle I've changed to `app` and `api` to differentiate each type of application. What do you think? If you think that there's the need for `camunda`, maybe we can use `.camunda.app` and `.camunda.api`